Hakawai/Mentions/HKWMentionsCreationStateMachine.m

```diff
@@ -365,7 +365,9 @@ - (void)mentionCreationStartedWithPrefix:(NSString *)prefix
                    usingControlCharacter:(BOOL)usingControlCharacter
                         controlCharacter:(unichar)character
                                 location:(NSUInteger)location {
-    if (self.networkState != HKWMentionsCreationNetworkStateQuiescent) {
+    // Because we are supporting insertion of strings (including valid mention-strings) via non-english keyboards,
+    // we should be able to start a new mention even without being in a quiescent state
```

Comment:
what are the implications of allowing this? not super familiar w/ hakawai's state machine

Reply:
there weren't any comments or logged errors, so I don't fully know :/ But this state machine runs on a whole list of assumptions that are not entirely clear

From my understanding, since there wasn't a comment or a logged error, this was just a return that confirmed an assumption made - that quiescent would always be the state here. This isn't always the case with japanese text replacement, for example. So, obviously after further testing, I intend to remove this check once the lix is ramped and update the comments/assumptions here

Reply:
ok sounds good, ya whatever we can do to make things clearer for the future will be great